packages/teams-js/src/public/webStorage.ts

```diff
@@ -18,10 +16,6 @@ export namespace webStorage {
    */
   export function isWebStorageClearedOnUserLogOut(): boolean {
     ensureInitialized();
-    // return true as web storage is always cleared on desktop.
```

Comment:
am I correct that this function only returns whether this is supported now?  if so, maybe we should change the function name to `isWebStorageSupported`